terrain/fixed_lod/voxel_terrain.cpp

```diff
@@ -1312,8 +1249,8 @@ void VoxelTerrain::refresh_voxel_viewer_data(uint32_t viewer_id) {
 
 		if (require_notifications && network_peer == viewer_id) {
 			const Box3i &data_box = viewer.state.data_box;
-			data_box.for_each_cell([this, &viewer, require_notifications](Vector3i bpos) { //
-				view_data_block(bpos, viewer.id, require_notifications);
+			data_box.for_each_cell([this, &viewer, require_notifications](Vector3i bpos) {
+				//view_data_block(bpos, viewer.id, require_notifications);
```

Comment:
@jeronimo-schreyer This method does not exist anymore in the Voxel module, can you make this method compatible with the latest Voxel module?